src/style.css

```
@@ -1 +1,63 @@
-/* Styles go here */
+.body {
+  font-family: Roboto, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  /* padding: 0 50px; */
```

```suggestion
  padding: 0 50px;
```
Не потрібно було це закоментовувати. Якраз так і має бути - в хедера по бокам падінги у 50 пікселів